README.md

```diff
@@ -1,3 +1,35 @@
 # REDCap to Harvest
 
 Dockerfile and tools for deriving Harvest applications from REDCap projects.
+
+Installation and Setup
+----------------------
+
+This tool runs using Docker. Install it if you don't have it:
+
+[Install Docker](http://docs.docker.com/installation/) 
+
+If you are using Mac, use boot2docker to run Docker.
+
+In order to run the container, you need to place your data dictonary and data file in a folder called Input. Name the data dictonary and data file "metadata.csv" and "data.csv", respectively.
```

Comment:
Thinking about this some more, it would be nice to *not* have to rename just for the container. We can define environment variables that can be used to specify the file name relative to the `/input` directory. Add the following to the Dockerfile:

```docker
ENV METADATA_FILE metadata.csv
ENV DATA_FILE data.csv
```

And then in the `start` script, replace the hard-coded names with the respective environment variables. This enables a person to do this:

```
docker run -e METADATA_FILE=some-meta-file.csv -e DATA_FILE=some-data.csv ...
```